lntest/itest/assertions.go

```diff
@@ -1744,23 +1687,20 @@ func sendAndAssertFailure(t *harnessTest, node *lntest.HarnessNode,
 	defer cancel()
 
 	stream, err := node.RouterClient.SendPaymentV2(ctx, req)
-	if err != nil {
-		t.Fatalf("unable to send payment: %v", err)
-	}
+	require.NoError(t.t, err, "unable to send payment")
 
 	result, err := getPaymentResult(stream)
-	if err != nil {
-		t.Fatalf("unable to get payment result: %v", err)
-	}
+	require.NoError(t.t, err, "unable to get payment result")
 
-	if result.Status != lnrpc.Payment_FAILED {
-		t.Fatalf("payment was expected to fail, but succeeded")
-	}
+	require.Equal(
+		t.t, result.Status, lnrpc.Payment_FAILED,
```

Comment:
nit: expected result needs to come first for proper error message. Same in the `Equal` case below.

Reply:
Fixed!